src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -443,11 +439,14 @@ public function close()
 	 */
 	public function __construct(array $options)
 	{
-        if (isset($options['log4php.properties'])) {
-            \Logger::configure($options['log4php.properties']);
-        }
-        $this->_logger = \Logger::getLogger('Pami.ClientImpl');
-	    $this->_host = $options['host'];
+		$this->setLogger(
+			new Log4phpAdapter(
```

Comment:
Let's call it "psr3_logger" so it is perfectly clear the kind of object expected